packages/styles/src/inputs/_inputs.scss

```diff
@@ -0,0 +1,110 @@
+@import "../core/_dev";
+
+.as-input,
+.as-textarea {
+  box-sizing: border-box;
+  width: 100%;
+  min-height: 32px;
+  padding: 6px 8px;
+  border: 0;
+  border-radius: 4px;
+  background: var(--as-color-ui-01, #{$color-ui-01});
+  box-shadow: inset 0 0 0 1px var(--as-color-ui-04, #{$color-ui-04});
+  color: var(--as-color-type-01, #{$color-type-01});
+  font: $font-body;
+
+  &::-webkit-input-placeholder {
```

Comment:
can we delegate this browser-specific selectors in a postcss plugin?

Reply:
Great idea! It seems like we can use `::placeholder` and it takes care of it :D

![screen shot 2018-08-02 at 13 04 45](https://user-images.githubusercontent.com/905225/43580149-ac2e15e0-9654-11e8-9f3d-e747347a7120.png)